Framework/HistogramData/src/Interpolate.cpp

```diff
@@ -128,7 +128,7 @@ void interpolateYCSplineInplace(const Mantid::HistogramData::Histogram &input,
       }
     }
   }
-  double xsMaxEpsilon = *(std::max_element(xs.begin(), xs.end())) * std::numeric_limits<double>::epsilon();
+  double xsMaxEpsilon = *(std::max_element(xs.begin(), xs.end()))*std::numeric_limits<double>::epsilon();
```

Comment:
This re-format seems like an odd choice. I would have expected it to like the spaces around the asterisk 🤔

Reply:
I'm not going to worry about this one